src/bank.rs

```diff
@@ -61,12 +61,27 @@ impl BankKeeper {
             .map_err(Into::into)
     }
 
-    // this is an "admin" function to let us adjust bank accounts
     fn get_balance(&self, bank_storage: &dyn Storage, account: &Addr) -> AnyResult<Vec<Coin>> {
         let val = BALANCES.may_load(bank_storage, account)?;
         Ok(val.unwrap_or_default().into_vec())
     }
 
+    fn get_supply(&self, bank_storage: &dyn Storage, denom: String) -> AnyResult<Coin> {
+        let supply: Uint128 = BALANCES
+            .range(bank_storage, None, None, Order::Ascending)
+            .map(|a| a.unwrap().1)
```

Comment:
I'd still like to get rid of this `unwrap`. There's multiple ways to do it, but I'd just collect into a `StdResult<Vec<_>>` and continue with that:
`BALANCES.range(...).collect::<StdResult<Vec<_>>>()?.map(...).fold(...)`.

Reply:
Cool, will do.